index.html

```diff
@@ -1379,6 +1379,16 @@ <h2>
             <td>
 Secp256k1 public key values MUST either be encoded as a JWK or be encoded as the
 raw 32-byte public key value in Base58 Bitcoin format using the
+<code>publicKeyBase58</code> property if they are not encoded in JWK format.
```

Comment:
```suggestion
<code>publicKeyBase58</code> property.
```
The trailing clause is now redundant.